docker-compose.yml

```diff
@@ -93,6 +93,24 @@ services:
       - ./api/docker/caddy/Caddyfile:/etc/caddy/Caddyfile:ro
       - ./api/public:/srv/api/public:ro
 
+  http-cache:
+    image: varnish:7.5.0
+    container_name: 'ecamp3-http-cache'
+    depends_on:
+      - caddy
+    volumes:
+      - ./api/docker/varnish/vcl/:/etc/varnish/:ro
+    tmpfs:
```

Comment:
I thought tmpfs only works on Linux..? Did you test this on a Mac?

Reply:
Thanks for the hint. I just copied this over from https://www.varnish-software.com/developers/tutorials/running-varnish-docker/

If I see correly, this is more of a performance tweak that shouldn't really be relevant for local development. Will remove.

Reply:
Removed with https://github.com/ecamp/ecamp3/pull/3610/commits/d7212f3555ea824a310de90e11c34cc12d206110